### ISSUE TITLE
Switch to playwright E2E testing ? 

### ISSUE THREAD

**Issue by themosaeed** [benawad/dogehouse#2354]
Are you concediring playwright E2E testing ? I can help setting this up or implementing the already done with cypress in playwright. If there is a team that is doing this I would love to join.

**Comment by nadirabbas** [comment]
Isn't playwright just an alternative?

**Comment by themosaeed** [comment]
> Isn't playwright just an alternative?

actually not, cypress has some limitations that playwright can address and in my opinion, is a better well-rounded option suited more for applications like dogehouse

**Comment by goldyydev** [comment]
I totally agree with this. Playwright just appears to be more flexible (to me)

@benawad might wanna take a look so this PR won't hang

**Comment by jobveldhuis** [comment]
I think if you feel like we need to switch to Playwright, you will at least need to explain why we need to switch and what it would bring the project, short and/or long term. This would be the same as starting a discussion stating: *we should switch to Nuxt instead of Next*, without giving **any** reason and/or arguments about why this change would be good.
